examples/util/base.js

```diff
@@ -251,11 +256,14 @@ const ssdlite_mobilenetv2_tflite = {
   preOptions: {
     mean: [127.5, 127.5, 127.5],
     std: [127.5, 127.5, 127.5],
-  }
-};
+  },
+  intro: 'MobileNet V2 with SSDLite. This model is faster than Mobilenet V1 SSD at the same accuracy.',
```

Comment:
![image](https://user-images.githubusercontent.com/40754037/53550168-04d63080-3b71-11e9-8cbc-8845ca1be307.png)
![image](https://user-images.githubusercontent.com/40754037/53550198-191a2d80-3b71-11e9-83b3-919f7f7cdaf0.png)
Compared with SSD Mobilenet V2, SSDLite Mobilenet V2 is much faster, and almost don't lose the accuracy.

Reply:
intro: 'MobileNet V2 with SSDLite, which is an upgraded version of SSD MobileNet V2. Compared with SSD Mobilenet V2, SSDLite Mobilenet V2 is much faster, and almost have no loss of the accuracy. '